README.md

```diff
@@ -30,76 +41,88 @@ $ helm install ohmyglb ohmyglb/ohmyglb
 See [values.yaml](https://github.com/AbsaOSS/ohmyglb/blob/master/chart/ohmyglb/values.yaml)
 for customization options.
 
-### Local Playground
-
-####  Deploy local cluster
-
-```sh
-$ make deploy-local-cluster
+### Local Playground Install
+
+#### Environment prerequisites
+
+ - [install **GO 1.14**](https://golang.org/dl/)
+ 
+ - [install **GIT**](https://git-scm.com/downloads)     
+    
+ - install **gnu-sed** if you don't have it  
+    - If you are on a Mac, install sed by Homebrew
+    ```shell script
+    brew install gnu-sed 
+    ``` 
+   
+ - [install **Docker**](https://docs.docker.com/get-docker/)
+    - ensure you are able to push/pull from your docker registry
+    - to run multiple clusters reserve 8GB of memory  
+    
+      ![](docs/images/docker_settings.png)
+      <div>
+        <sup>above screenshot is for <strong>Docker for Mac</strong> and that options for other Docker distributions may vary</sup>
+      </div>
+
+ - [install **Kubectl**](https://kubernetes.io/docs/tasks/tools/install-kubectl/) to operate clusters
+     
+ - [install **Helm3**](https://helm.sh/docs/intro/install/) to get charts
+ 
+ - [install **kind**](https://kind.sigs.k8s.io/) as tool for running local Kubernetes clusters
+    - follow https://kind.sigs.k8s.io/docs/user/quick-start/
+ 
+
+
+#### Running project locally
+
+To spin-up a local environment using two Kind clusters and deploy a test application to both clusters, execute the command below: 
+```shell script
+make deploy-full-local-setup 
 ```
-Creates local [kind](https://github.com/kubernetes-sigs/kind) cluster
-with several workers for more realistic setup.
 
-#### Deploy local ingress
-
-```sh
-$ make deploy-local-ingress
-```
-Creates local nginx ingress with deployment config similar to Rancher.
-IP addresses of workers will be exposed as Ingress addresses.
-It will create proper environment for Ohmyglb testing
 
-#### Deploy gslb operator
+#### Verify installation
+If local setup runs well, check if clusters are correctly installed
 
-```sh
-$ make deploy-gslb-operator
+```shell script
+kubectl cluster-info --context kind-test-gslb1 && kubectl cluster-info --context kind-test-gslb2
```

Comment:
just one last thing from my side. `kubectl cluster-info` will verify only basic cluster availability.
Meanwhile to proceed to next step with `dig` based DNS queries we need to verify that ohmyglb deployment is ready.
As per helm chart post install output it can be achieved by:
```
1. Watch etcd cluster start
  $ kubectl get pods -l etcd_cluster=etcd-cluster --namespace ohmyglb -w

2. Confirm etcd cluster is healthy
  $ kubectl run --rm -i --tty --env="ETCDCTL_API=3" --env="ETCDCTL_ENDPOINTS=http://etcd-cluster-client:2379" --namespace ohmyglb etcd-test --image quay.io/coreos/etcd --restart=Never -- /bin/sh -c 'watch -n1 "etcdctl  member list"'
```
I suggest to use `2.` in documentation and provide expected output like
```
193d78f5660d9589, started, etcd-cluster-qngdnl2gk4, http://etcd-cluster-qngdnl2gk4.etcd-cluster.ohmyglb.svc:2380, http://etcd-cluster-qngdnl2gk4.etcd-cluster.ohmyglb.svc:2379
2b980bfa84d157b4, started, etcd-cluster-bgn7cxc99d, http://etcd-cluster-bgn7cxc99d.etcd-cluster.ohmyglb.svc:2380, http://etcd-cluster-bgn7cxc99d.etcd-cluster.ohmyglb.svc:2379
5a69362add606d9a, started, etcd-cluster-q7zjfck7p9, http://etcd-cluster-q7zjfck7p9.etcd-cluster.ohmyglb.svc:2380, http://etcd-cluster-q7zjfck7p9.etcd-cluster.ohmyglb.svc:2379
```

Reply:
@ytsarev, Hi Yury, sure. I can make something like 

 ```shell script
make healthy-full-local-setup
```

But there is another issue. It takes cca 30 min until clusters are fully initialized . (My local docker has 9GB + 6CPU reserved). If I run helthy check immediatlly, it just enqueues new pods and wait. Maybe we should inform users to wait before verification until all pods are up . 

```shell script
NAME                                                          READY   STATUS              RESTARTS   AGE
etcd-cluster-5gjfhjnnpn                                       0/1     Init:0/1            0          22m
external-dns-86b4c8db9-rgb6s                                  1/1     Running             0          25m
nginx-ingress-controller-5jh4c                                0/1     ContainerCreating   0          24m
nginx-ingress-controller-5kflw                                1/1     Running             0          24m
nginx-ingress-controller-rm6ls                                1/1     Running             0          24m
nginx-ingress-default-backend-5b967cf596-6xpsx                1/1     Running             0          24m
ohmyglb-576d6b887f-hsj28                                      1/1     Running             0          25m
ohmyglb-coredns-dd47b4df9-bz8f6                               1/1     Running             0          25m
ohmyglb-etcd-operator-etcd-backup-operator-585f68d555-6lzqc   1/1     Running             0          25m
ohmyglb-etcd-operator-etcd-operator-59bcfc5c46-qxq2m          1/1     Running             0          25m
ohmyglb-etcd-operator-etcd-restore-operator-98899587f-8k9fs   1/1     Running             0          25m


Every 2.0s: kubectl get pods -n test-gslb                                                                                                                                                                           C02XR6YRJGH6: Sun Apr 19 15:56:58 2020

NAME                                READY   STATUS              RESTARTS   AGE
frontend-podinfo-7998d695bd-4vs69   0/1     ContainerCreating   0          26m
frontend-podinfo-7998d695bd-6hsj7   1/1     Running             0          26m
healthy-app-64fc6c4449-wrb66        0/1     ContainerCreating   0          26m


```

Reply:
@kuritka it's clearly something wrong with your setup, whole initialization usually takes 2-3 minutes. 
See https://github.com/AbsaOSS/ohmyglb/runs/598134574?check_suite_focus=true , ohmyglb deployment step took 2m 41s even in limited github actions environment.
There are embedded `kubectl wait` checks in https://github.com/AbsaOSS/ohmyglb/pull/91 which I am using after deployment so no need of separate make target.
Please just document the process how to smoke test the ohmyglb setup - checking the etcd cluster state and dig queries(you have them below)